libtiledbvcf/src/CMakeLists.txt

```diff
@@ -135,20 +135,43 @@ endif()
 add_library(tiledbvcf SHARED
   $<TARGET_OBJECTS:TILEDB_VCF_OBJECTS>
 )
+if (WIN32)
+  # tiledb_vcf_unit ref's symbols that are not explicitly part of the exported public interface.
+  # Since Windows/MSVC does not make all symbols in a shared library visible,
+  # Use a cmake feature on a target specifically for tiledb_vcf_unit to export all symbols.
+  add_library(tiledbvcf4test SHARED
```

Comment:
Couldn't this be done with an object library instead? That would avoid the installation questions below entirely.

Reply:
Assuming you mean creating an object library that links directly into tiledb_vcf_unit and into the tiledbvcf.dll...
That wouldn't be quite the same - if there are any (memory or other) 'boundary' issues that might be exercised by tests, linking an object library in directly would eliminate that boundary.
(Example of this is the htslib alloc/free of memory across boundary - don't know if this extends into any of the tiledbvcf provided apis or not, or whether tiledbvcf itself might provide anything similarly problematic...)